src/game_inventory.cpp

```diff
@@ -482,6 +511,52 @@ class comestible_inventory_preset : public inventory_selector_preset
             return dummy;
         }
 
+        const std::string get_time_left_rounded( const item_location &loc ) {
+            const item *item = loc.get_item();
+            const double relative_rot = item->is_food_container() ? item->contents.front().get_relative_rot() :
+                                        item->get_relative_rot();
```

Comment:
This can (or rather should) be replaced with `get_comestible_item`.

Reply:
Sounds good to me, thanks!